olive/workflows/run/run.py

```diff
@@ -76,6 +76,12 @@ def dependency_setup(config):
             "TorchTRTConversion": EXTRAS.get("torch-tensorrt"),
         },
     }
+    ORT_PACKAGE_TO_EP_MAPPING = {
```

Comment:
it seems this map is some kind of duplicated with https://github.com/microsoft/OLive/blob/8dc23995537b5db607a4e5b9a43859095134a885/olive/systems/utils.py#L66

Reply:
I saw that mapping too but wasn't sure if we can reuse this since the mapping is reverse. and the reverse is not 1 to 1.

Reply:
This mapping doesn't exist anymore. Found a better way to find the package.